### PR TITLE
Add gocritic importShadow linter and fixes

### PR DESCRIPTION
**Pull request by saschagrunert**
This commit fixes issues related to the `importShadow` linter, which
detects when imported package names shadowed in the assignments.

### CODE REVIEW

File: lib/container_server.go [hunk 1]
```diff
@@ -674,11 +674,11 @@ type containerServerState struct {
...
 // AddContainer adds a container to the container state store
 func (c *ContainerServer) AddContainer(ctr *oci.Container) {
-	sandbox := c.state.sandboxes.Get(ctr.Sandbox())
-	if sandbox == nil {
+	sb := c.state.sandboxes.Get(ctr.Sandbox())
+	if sb == nil {
 		return
 	}
-	sandbox.AddContainer(ctr)
+	sb.AddContainer(ctr)
```
Comment (TomSweeneyRedHat) [inline lib/container_server.go]: I probably would have gone 'newSandbox", but shrug.

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: lib/container_server.go (hunk 1 above)
Comment (saschagrunert) [reply lib/container_server.go]: I renamed it :)

**Pull request merged**